### DISCUSSION TITLE
请问本地 docker 部署，surge 订阅，所有节点都是失败是我自己的配置问题吗？

### DISCUSSION THREAD

**Discussion by TheFoolzzZ** [vvbbnn00/WARP-Clash-API#125] [Q&A] [answered]
rt
<img width="280" alt="1709444777209" src="https://github.com/vvbbnn00/WARP-Clash-API/assets/37650571/add7afee-d994-45f0-81c3-2d2e33834d0e">

**Comment by vvbbnn00** [comment, marked as answer]
surge目前有点问题，通过外部代理列表导入节点的话，就会变成这样，建议先使用完整配置文件
